src/containers/home/invest/component.tsx

```diff
@@ -8,12 +8,54 @@ import AnimatedScroll from 'containers/home/animated-scroll';
 import Wrapper from 'containers/wrapper';
 
 import Button from 'components/button';
+import Carousel from 'components/carousel';
 import Icon from 'components/icon';
 
 import CHECK_SVG from 'svgs/ui/check.svg?sprite';
+import CHEVRON_RIGHT from 'svgs/ui/chevron-right.svg?sprite';
 
 const Invest = () => {
   const { data: areasData } = useAreas();
+  const [slide, setSlide] = useState(0);
+
+  const formatedAreasData = useMemo(() => {
+    const perChunk = 6; // items per chunk
+
+    const panels = areasData.reduce((all, one, i) => {
+      const ch = Math.floor(i / perChunk);
+      all[ch] = [].concat(all[ch] || [], one);
+      return all;
+    }, []);
+
+    return panels.map((areas, i) => {
+      return {
+        id: `panel-${i}`,
+        content: (
+          <ul key={i} className="space-y-5">
+            {areas.map((area) => (
+              <li key={area.id} className="flex space-x-2.5 text-xl text-white">
+                <div className="flex items-center justify-center w-5 h-5 mt-1 bg-white rounded-full shrink-0">
+                  <Icon className="w-3 h-3 text-blue-0" icon={CHECK_SVG} />
+                </div>
+                <span>{area.name}</span>
+              </li>
+            ))}
+          </ul>
+        ),
+      };
+    });
+  }, [areasData]);
+
+  const handleOnNextClick = useCallback(() => {
+    const length = formatedAreasData?.length;
+
+    if (slide === length - 1) {
+      setSlide(0);
+    } else {
+      const slideAdd = slide + 1;
+      setSlide(slideAdd);
```

Comment:
I think you should use 
```suggestion
      setSlide((prevSlide) => {
        if (prevSlide + 1 === lenght) {
          return 0;
        }
        
        return prevSlide + 1;
      });
```

By using this approach, you won't have issues with multiple clicks on the button. I think you should do the same with the onChange method.